.github/workflows/cifuzz.yml

```
@@ -0,0 +1,36 @@
+name: CIFuzz
+true:
```

```suggestion
on:
```

this should be "on" to be consistent with the other workflows